samples/msal-react-samples/react-router-sample/package.json

```diff
@@ -5,8 +5,10 @@
   "dependencies": {
     "@azure/msal-browser": "file:../../../lib/msal-browser",
     "@azure/msal-react": "file:../../../lib/msal-react",
-    "@material-ui/core": "^4.11.2",
-    "@material-ui/icons": "^4.11.2",
+    "@emotion/react": "^11.10.5",
```

Comment:
>    "@azure/msal-browser": "file:../../../lib/msal-browser",
>    "@azure/msal-react": "file:../../../lib/msal-react"

Can we use published versions as default? (also for `react `and `react-dom` below)